dependencies.toml

```diff
@@ -3,7 +3,7 @@
 #     If its classes are exposed in Javadoc, update offline links as well.
 #
 [versions]
-armeria = "1.29.4"
+armeria = "1.30.0"
```

Comment:
Do you want to update dependencies in a separate PR?

Reply:
It was updated to use the new API in Armeria. The dependency upgrade PR also includes Armeria updates.